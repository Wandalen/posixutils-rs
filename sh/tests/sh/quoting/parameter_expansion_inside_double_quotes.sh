var=value
echo "$var"
echo "var=$var"
echo "${var}"
echo "var=${var}"
echo "${#var}"
echo "len(var)=${#var}"