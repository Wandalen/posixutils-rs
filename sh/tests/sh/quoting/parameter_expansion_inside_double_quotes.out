value
var=value
value
var=value
5
len(var)=5
